lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/data/collectionData.py

```diff
@@ -0,0 +1,80 @@
+from typing import AnyStr, Sequence
+
+from .stringListData import StringListData
+
+try:
+    from PySide6.QtCore import QObject, Signal  # type: ignore
+except ImportError:
+    from PySide2.QtCore import QObject, Signal  # type: ignore
+
+class CollectionData(QObject):
+    dataChanged = Signal()
+
+    def __init__(self):
+        super().__init__()
+
+    # Item picking
+
+    def canPick(self) -> bool:
+        '''
+        Return if there is a UI to pick items.
+        '''
+        return False
+    
+    def pick(self) -> Sequence[AnyStr]:
+        '''
+        Pick items and return the sequnce of picked items.
+        '''
+        return []
```

Comment:
What's the semantics about data being able to pick??? Only some kind of host can pick - data can only change. 

And if this would make syntactically sense, then canDrop would be missing (what seems to be still in host).

Reply:
I was worrying that picking in the future might need to know about the data (collection, prim or any others) that would affect it. But for now it doesn't, I can remove it and leave it in Host for now.

Reply:
Looking at the code, the picking does need to know about the data to get the stage. I guess I will expose the stage in the data for now.